applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc

```diff
@@ -6,15 +6,17 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+[role="_abstract"]
 You can install Helm charts on an {product-title} cluster using the following methods:
 
 * The CLI.
 * The *Developer* perspective of the web console.
 
 The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat OpenShift Helm chart repository. For a list of the charts, see link:https://charts.openshift.io/index.yaml[the Red Hat `Helm index` file].
 
-As a cluster administrator, you can add multiple Helm chart repositories, apart from the default one, and display the Helm charts from these repositories in the *Developer Catalog*.
+As a cluster administrator, you can add multiple Helm chart repositories, separate from the default one, and display the Helm charts from these repositories in the *Developer Catalog*.
```

Comment:
Would it make sense to s/you can add multiple Helm chart repositories/you can add multiple cluster-scoped and namespace-scoped Helm chart repositories ?